tariff/awattar.go

```diff
@@ -61,17 +62,27 @@ func (t *Awattar) Run() {
 	}
 }
 
-func (t *Awattar) IsCheap() bool {
+func (t *Awattar) CurrentPrice() (float64, error) {
 	t.mux.Lock()
 	defer t.mux.Unlock()
 
 	for i := len(t.data) - 1; i >= 0; i-- {
 		pi := t.data[i]
 
 		if pi.StartTimestamp.Before(time.Now()) && pi.EndTimestamp.After(time.Now()) {
-			return pi.Marketprice/10 <= t.cheap // Eur/MWh conversion
+			return pi.Marketprice / 10, nil
 		}
 	}
 
-	return false
+	return 0, errors.New("unable to find current awattar price")
+}
+
+func (t *Awattar) IsCheap() bool {
+	price, err := t.CurrentPrice()
+
+	if err != nil {
```

Comment:
return price <= t.cheap, err

Reply:
Ist aus den tariff Implementierungen rausgenommen und das Error Handling ist nun in `site`.